modules/nixos/buildbot.nix

```diff
@@ -14,6 +14,15 @@
     "http://localhost:8011/metrics"
   ];
 
+  systemd.services.buildbot-master.serviceConfig = {
+    Restart = "on-failure";
```

Comment:
I can probably upstream this.

Reply:
Thanks.

Reply:
Seems this needs to be always instead of on-failure as the service exits successfully. I started this PR with always and switched to on-failure, I thought I'd checked it but must have misremembered. Shall I send a PR to change it to always in buildbot-nix?